src/api/types/cron/index.ts

```diff
@@ -1,3 +1,35 @@
+import * as cron from 'node-cron';
+import { SuccessResponse } from '../generic';
+import { Context } from '.keystone/types'; // eslint-disable-line
+
+export type CronScheduleFunc = Parameters<typeof cron.schedule>[1];
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type CronScheduleFuncParameters = Parameters<Extract<CronScheduleFunc, Function>>[0];
+
+/**
+ * The `now` parameter is provided here as
+ * node-cron's schedule function expects tasks to take this parameter.
+ * eslint disable required as variables need to be defined for cron jobs (but not used in this file)
+ */
+// eslint-disable-next-line no-unused-vars
+export type CronSchedulerJobTask = (context: Context, now: CronScheduleFuncParameters) => Promise<SuccessResponse>;
+
+export type CronSchedulerJob = {
+  /**
+   * A cron expression representing when the job will run
+   */
+  cronExpression: string;
+  /**
+   * A description of the job
+   */
+  description: string;
+  /**
+   * The task to be run by the job
+   */
+  task: CronSchedulerJobTask;
+};
+
```

Comment:
Lots of eslint disabling in here?

Reply:
they are required.  Same in DTFS.  ESLint does not allow Function declaration but it is required.  Also the variables need to be declared but can't be used in the types file so need a disable too